src/main/java/racinggame/core/GameInitializer.java

```diff
@@ -0,0 +1,29 @@
+package racinggame.core;
+
+import java.util.ArrayList;
+import java.util.List;
+import racinggame.exception.InvalidCarRegistrationException;
+
+public class GameInitializer {
+
+    public static final String DELIMITER = ",";
+
+    public static List<Car> initCars(String carNames) {
+        List<Car> cars = new ArrayList<>();
+        String[] carNameArray = carNames.split(DELIMITER);
+        validCarNames(carNameArray);
+
+        for (String carName : carNameArray) {
+            cars.add(new Car(carName));
+        }
+
+        return cars;
+    }
+
+    private static void validCarNames(String[] carNameArray) {
+        if (carNameArray.length <= 0) {
```

Comment:
```suggestion
        if (carNameArray.length == 0) {
```

`length` 값은 음수일 수 없겠죠~